src/docs/product/issues/issue-details/performance-issues/endpoint-regression/index.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: "Endpoint Regression"
+sidebar_order: 41
+description: "Learn more about Endpoint Regression issues and how to diagnose and fix them."
+---
+
+<Include name="feature-available-for-user-group-early-adopter.mdx" />
+
+Endpoint Regression issues are a generic class of problems where the duration of a transaction increases over time and degrades application performance. Sentry proactively monitors common endpoints out of the box and reports any possible regressions, grouping them as Endpoint Regression issue types.
+
+## Detection Criteria
+
+The detector for performance issues periodically checks the 95th percentile transaction duration of the most common endpoints in your project. When a significant increase in the p95 value is detected and has been sustained for some time, a regression issue is created.
+
+## Regression Evidence
+
+To find additional info, go to the Issues page and click on the Endpoint Regression issue you're interested in. In the top section of the "Details" tab, you'll find the following:
+
+- **Endpoint Name:** The name of the transaction that has regressed.
+- **Change in Duration:** The value of absolute and relative change in duration.
+- **Approximate Start Time:** The approximate time when the regression occurred.
+
+## Charts
+
+The below chart shows the p95 transaction duration as well as the transaction throughput over a period of up to 14 days before and 14 days after the regression was identified.
+
+![Endpoint Regression Issue Chart](endpoint-regression-chart.png)
```

Comment:
The throughput chart is moving to the side bar. Can we remove mentions of it from here and update the screenshot once the changes are in?